manage.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+
+import code
+from os.path import join, dirname
+import subprocess
+import sys
+
+import click
+from dotenv import load_dotenv
+import pytest
+
+
+def load_app():
+    # Environment variable MUST be set before importing the app
+    dotenv_path = join(dirname(__file__), '.env')
+    load_dotenv(dotenv_path)
+
+    from tululbot import app
+    return app
+
+
+@click.group()
+def manage():
+    """Script to manage tasks outside the application itself."""
+    pass
+
+
+@manage.command()
+def shell():
+    """Run a shell with custom context."""
+    context = dict(app=load_app())
+    try:
+        from IPython import embed
+    except ImportError:
+        code.interact(local=context)
+    else:
+        embed(user_ns=context)
+
+
+@manage.command()
+def runserver():
+    """Run the application server."""
+    load_app().run()
+
+
+@manage.command()
+def test():
+    """Run the tests."""
+    # Environment variable MUST be set before importing the app
+    dotenv_path = join(dirname(__file__), 'tests', '.env')
+    load_dotenv(dotenv_path)
+
+    sys.exit(pytest.main([]))
+
+
+@manage.command()
+def lint():
+    """Run the linters."""
+    sys.exit(subprocess.call(['flake8']))
+
+
+@manage.command()
+def check():
```

Comment:
I tried that, but it doesn't work. It always fails with missing argument error. I think this has something to do with `@manage.command()` decorator. And notice that `test()` and `lint()` call `sys.exit()`, not returning the exit code. Thus we can't call them inside `check()`; one of them will never get called.

Of course we can get around this but I think the resulting code will be quite ugly. I prefer it this way.